build.gradle

```diff
@@ -27,4 +27,9 @@ checkstyleMain {
 }
 
 test.finalizedBy("checkstyleMain")
-test.finalizedBy("jacocoTestReport")
\ No newline at end of file
+test.finalizedBy("jacocoTestReport")
+
+task runBanckocr (dependsOn: 'classes', type: JavaExec) {
```

Comment:
This file seems to be having a conflict that prevents merge. 

It is not necessary to add this task so maybe the easiest thing you can do is just revert changes in this file. Your code should be mainly tested by unit tests, so this task would be optional.

Reply:
fixed